doc/classes/RenderingDevice.xml

```diff
@@ -590,6 +590,7 @@
 		</method>
 		<method name="screen_get_framebuffer_format" qualifiers="const">
 			<return type="int" />
+			<param index="0" name="screen" type="int" default="0" />
 			<description>
 				Returns the screen's framebuffer format.
```

Comment:
Since you added a screen parameter it may be worth being more explicit.
```suggestion
				Returns the framebuffer format of the given screen.
```

Reply:
Agreed.